### DISCUSSION TITLE
Saleor 3.0 - Store information Country field error and dynamic routes

### DISCUSSION THREAD

**Discussion by dialformonkey** [saleor/saleor#8387] [Q&A] [answered]
So i've setup my Saleor instance and the dashboard on Vercel. Ive run into dynamic routes not working on refresh of pages as well as not being able to select a country from the configuration for Company Information.

Anyone else dealing with this?

<img width="1193" alt="Screenshot 2021-10-18 at 1 59 51 AM" src="https://user-images.githubusercontent.com/62189575/137644797-d482a007-ab27-455d-bb67-aa75231dc4a5.png">

**Comment by karolkielecki** [comment, marked as answer]
@dialformonkey Hi! Are you using the newest beta version? More info will be helpful for us to reproduce the issue.

**Reply by dialformonkey** [reply]
Yes. I am using the  latest beta version of both core and dashboard

**Reply by dialformonkey** [reply]
I've managed to fix this. However I still have not found a solution for the dynamic route refresh 404 error.

**Reply by karolkielecki** [reply]
Could you please share how did you fix it? Regarding dynamic routes--also need more details, e.g. screenshots, and please check dev console to see if there are any errors there.

**Reply by dialformonkey** [reply]
Turns out my saleor core was 2 commits behind and proper migrations were not done.

For the route pathname not working it was just a simple vercel rewrite rule that needed to be added.
